getting-started/data-setup.rst

```diff
@@ -46,149 +44,91 @@ It may simply be that you're working in a brand new shell.
+.. code-block:: python
 
-Ingesting calibrations into the Butler repository
-=================================================
+   registry = butler.registry
+   for col in registry.queryCollections():
+       print(col)
+   for ref in registry.queryDatasets('raw', collections='', instrument='HSC'):
+       print(ref.full)
 
-Next, add calibration images (such as dark, flat, and bias frames) associated with the raw data:
-
-.. code-block:: bash
+Notes on processing
+===================
 
-   ln -s $TESTDATA_CI_HSC_DIR/CALIB/ DATA/CALIB
+The intention of this set of introductory recipes is to give you a realistic sense of how data is processed using the LSST Science Pipelines.
+That includes taking raw images all the way through to coaddition and forced photometry.
+Though the starting repository is small, a significant amount of processing needs to be done to produce all the datasets needed for downstream processing.
+This means that some steps can be quite time consuming and you should be prepared to wait or perhaps run things overnight if you intend to follow these examples line by line.
 
-.. note::
+The most time consuming steps are:
 
-   In general, you can use the :command:`ingestCalibs.py` command-line task to ingest calibrations into a Butler repository.
-   For this tutorial, we've taken a shortcut by manually symlinking pre-structured calibrations from the `testdata_ci_hsc`_ package.
+- Single frame processing: 11 hours
+- Warping the images in preparation for coaddition: 90 minutes
+- Coaddition: 70 minutes
+- Coadd detection, deblending and measurement: 90 minutes
+- Forced photometry: 75 minutes
 
-Ingesting a reference catalog into the Butler repository
-========================================================
-
-The Pipelines use external stellar catalogs to refine the WCS and photometric calibration of images.
-`testdata_ci_hsc`_ includes a subset of the Pan-STARRS PS1 catalog that has been prepared as an astrometric and photometric reference catalog.
-Ingest that catalog into the Butler repository by creating a symlink:
-
-.. code-block:: bash
-
-   mkdir -p DATA/ref_cats
-   ln -s $TESTDATA_CI_HSC_DIR/ps1_pv3_3pi_20170110 DATA/ref_cats/ps1_pv3_3pi_20170110
-
-.. Processing tasks use these reference catalogs through configurations.
-.. The Pipelines will use this Pan-STARRS catalog by default 
-
-.. seealso::
-
-   Learn more about the PS1 reference catalog and how to use it with the LSST Science Pipelines in this `LSST Community forum topic <https://community.lsst.org/t/pan-starrs-reference-catalog-in-lsst-format/1572>`__.
-
-..
-   FIXME
-   We'll need to link to additional documentation on reference catalogs and their preparation.
-   Is manually linking a reference catalog our standard practice?
+These timings are all for a single serial thread.
+Some steps can be sped up significantly if you have access to more than one core.
```

Comment:
Single frame processing for example because surely that's massively parallel per chip so `-j4` will make a big difference.